docs/index.md

```diff
@@ -74,7 +74,12 @@ The following items are limitations with the current release of the Red Hat Clou
 
 To use the Red Hat Cloud Services provider inside your Terraform configuration you must meet the following:
 
-* Completed [the ROSA getting started](https://console.redhat.com/openshift/create/rosa/getstarted) requirements
+* You completed [the ROSA getting started](https://console.redhat.com/openshift/create/rosa/getstarted) requirements.
+* You generated an offline [OCM token](https://console.redhat.com/openshift/token/rosa). This token is generated through the Red Hat Hybrid Cloud Console. Do not share this token.
+* You have confirmed that you have [GoLang version 1.20 or newer](https://go.dev/doc/install) installed.
+* You have [Terraform version 1.4.6 or newer](https://developer.hashicorp.com/terraform/downloads) installed.
+* You may chose to use a [configured `*.tfvars` file](docs/terraform-vars.md). This file is a definition sheet for all of your variables.
+* You have created your [ROSA account roles](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/introduction_to_rosa/rosa-sts-about-iam-resources). To create the account roles using Terraform, see the [Account Roles Terraform example](https://github.com/terraform-redhat/terraform-provider-rhcs/tree/main/examples/create_account_roles/).
```

Comment:
This is prerequisites specific for the cluster creation step and not general prerequisite for using the provider